typed_python/PyConstDictInstance.cpp

```diff
@@ -234,6 +234,8 @@ PyObject* PyConstDictInstance::pyOperatorConcrete(PyObject* rhs, const char* op,
                 type()->addDicts(dataPtr(), w_rhs->dataPtr(), data);
             });
         } else {
+            if (!PyDict_Check(rhs))
```

Comment:
as written, we can't add two ConstDict's of different type, but we can add a ConstDict and a regular dict. We should allow adding anything with the mapping protocol to a ConstDict. However, I wouldn't fix this in this commit - let's file a ticket for it and come back to it.

Reply:
Ok, this has been done.